src/Data/Tree/Binary/Properties.agda

```diff
@@ -48,3 +51,14 @@ private
 map-id : ∀ (t : Tree N L) → map id id t ≡ t
 map-id (leaf x)     = refl
 map-id (node l v r) = cong₂ (flip node v) (map-id l) (map-id r)
+
+map-compose : ∀ {f₁ : N₁ → N₂} {f₂ : N → N₁} {g₁ : L₁ → L₂} {g₂ : L → L₁} (t : Tree N L) → map (f₁ ∘ f₂) (g₁ ∘ g₂) t ≡ map f₁ g₁ (map f₂ g₂ t)
+map-compose (leaf x) = refl
+map-compose (node l v r) = cong₂ (λ l r → node l _ r) (map-compose l) (map-compose r)
+
+map-cong : ∀ {f₁ f₂ : N → N₁} {g₁ g₂ : L → L₁} → (∀ n → f₁ n ≡ f₂ n) → (∀ l → g₁ l ≡ g₂ l) → (t : Tree N L) → map f₁ g₁ t ≡ map f₂ g₂ t
```

Comment:
If you look at [`Data.List.Properties`](http://agda.github.io/agda-stdlib/Data.List.Properties.html#3229),
we typically use `_≗_` in the definition of `map-cong`.

Reply:
Yes. I avoided this because other things in `Data.Tree.Binary.Properties` wrote it this way (`map-id`). Would it be better to convert all of them to this style or just the two I've added?